### PR TITLE
First pass at auth design and layout

### PR DESCRIPTION
**Pull request by creuter**
We don't need to worry about email and password fields because we're using Automatic's authentication system.

We'll revisit the skinning of everything in a few days. This basic styling is just to get an impression of "winter", while still keeping everything dark, slick and dashboardy.

This page still needs a logo and fonts, which will come in follow-up PRs.

screen shot 2015-11-11 at 3 54 45 pm

### CODE REVIEW

File: app/views/pages/auth.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <h2>What’s your driving style?</h2>
+      <p>The Automatic adapter plugs into the standard diagnostics port hidden
+      under the dash in your cars, to connect your car to a world of apps. If
+      you already have an Automatic account, sign in to see your driving
+      behavior. If you don't have one yet, take the quiz to see what archetype
+      you best fit!</p>
+    </div>
+
+    <div class="auth-actions">
+      <a href="#" class="button auth-button dummy">Sign in with Automatic</a>
```
Comment (creuter) [inline app/views/pages/auth.html]: I like using `dummy` classes for things that still need wiring up, especially in distributed teams where people will be rolling on and off.

They're searchable, and easy to style for highlighting in an audit before shipping.

**Comment by ColeTownsend** [comment]
looks sweet! code is fine too

File: app/views/pages/auth.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div class="auth-wrapper">
+  <div class="auth-container">
+    <div class="auth-description">
+      <h3>Archetypes</h3>
+      <h2>What’s your driving style?</h2>
+      <p>The Automatic adapter plugs into the standard diagnostics port hidden
+      under the dash in your cars, to connect your car to a world of apps. If
+      you already have an Automatic account, sign in to see your driving
+      behavior. If you don't have one yet, take the quiz to see what archetype
```
Comment (paulcsmith) [inline app/views/pages/auth.html]: Internally I think "archetype" is fine, but maybe something friendlier for the text. Like "take the quiz to see what type of driver you are"? Either way not a big deal

File: app/views/pages/auth.html (hunk 1 above)
Comment (paulcsmith) [inline app/views/pages/auth.html]: I've never done it before, but I'm down to give it a try!

File: app/views/pages/auth.html (hunk 2 above)
Comment (creuter) [inline app/views/pages/auth.html]: Good call, updating.

**Comment by paulcsmith** [comment]
The CSS looks good to me, but you might want a designer to take a look as well. The rest looks great!

**Pull request merged**